Name demangler broke Makefile builds

It appears that the addition of "utils/demangle.hpp" was not added to the Makefile builds. Cloning a fresh repository, and running "make" in "profiling/space-time-stack" results in:

	mpicxx -shared -O3 -g -fPIC -std=c++11 -Wall -Wextra -I/home/tvoskui/kokkos-tools/profiling/space-time-stack/ -I/home/tvoskui/kokkos-tools/profiling/space-time-stack//../../common/makefile-only -I/home/tvoskui/kokkos-tools/profiling/space-time-stack/../all -o kp_space_time_stack.so /home/tvoskui/kokkos-tools/profiling/space-time-stack/kp_space_time_stack.cpp
	/home/tvoskui/kokkos-tools/profiling/space-time-stack/kp_space_time_stack.cpp:34:10: fatal error: utils/demangle.hpp: No such file or directory
	   34 | #include "utils/demangle.hpp"
		  |          ^~~~~~~~~~~~~~~~~~~~
	compilation terminated.
	make: *** [kp_space_time_stack.so] Error 1

--- a/profiling/simple-kernel-timer/Makefile
+++ b/profiling/simple-kernel-timer/Makefile
@@ -1,12 +1,12 @@
 CXX=g++
-CXXFLAGS=-O3 -std=c++11 -g
+CXXFLAGS=-O3 -std=c++17 -g
 SHARED_CXXFLAGS=-shared -fPIC
 
 all: kp_kernel_timer.so kp_reader kp_json_writer
 
 MAKEFILE_PATH := $(subst Makefile,,$(abspath $(lastword $(MAKEFILE_LIST))))
 
-CXXFLAGS+=-I${MAKEFILE_PATH} -I${MAKEFILE_PATH}/../../common/makefile-only -I${MAKEFILE_PATH}../all
+CXXFLAGS+=-I${MAKEFILE_PATH} -I${MAKEFILE_PATH}/../../common/makefile-only -I${MAKEFILE_PATH}../all -I${MAKEFILE_PATH}../../common
 
 kp_reader: ${MAKEFILE_PATH}kp_reader.cpp kp_kernel_timer.so
 	$(CXX) $(CXXFLAGS) -o kp_reader ${MAKEFILE_PATH}kp_reader.cpp ${MAKEFILE_PATH}kp_shared.cpp 

--- a/profiling/space-time-stack/Makefile
+++ b/profiling/space-time-stack/Makefile
@@ -1,12 +1,12 @@
 CXX=mpicxx
-CXXFLAGS=-shared -O3 -g -fPIC -std=c++11 -Wall -Wextra
+CXXFLAGS=-shared -O3 -g -fPIC -std=c++17 -Wall -Wextra
 
 #Turn MPI support off:
 #CXXFLAGS += -DUSE_MPI=0
 
 MAKEFILE_PATH := $(subst Makefile,,$(abspath $(lastword $(MAKEFILE_LIST))))
 
-CXXFLAGS+=-I${MAKEFILE_PATH} -I${MAKEFILE_PATH}/../../common/makefile-only -I${MAKEFILE_PATH}../all
+CXXFLAGS+=-I${MAKEFILE_PATH} -I${MAKEFILE_PATH}/../../common/makefile-only -I${MAKEFILE_PATH}../all -I${MAKEFILE_PATH}../../common
 
 kp_space_time_stack.so: ${MAKEFILE_PATH}kp_space_time_stack.cpp
 	$(CXX) $(CXXFLAGS) -o $@ $<

